counterpartylib/lib/util.py

```diff
@@ -28,6 +29,9 @@
 D = decimal.Decimal
 b26_digits = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
 
+# subasset contain only characters a-zA-Z0-9.-_@!
+subasset_digits = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.-_@!'
```

Comment:
please make upper case as it's a constant (I see that `b26_digits` is lowercase too, that could be made upper case as well)